Source/CarthageKit/Project.swift

```diff
@@ -257,13 +271,13 @@ public final class Project {
 	///
 	/// Returns a signal which will send the URL to the repository's folder on
 	/// disk once cloning or fetching has completed.
-	private func cloneOrFetchDependency(project: ProjectIdentifier) -> ColdSignal<NSURL> {
+	private func cloneOrFetchDependency(project: ProjectIdentifier) -> SignalProducer<NSURL, CarthageError> {
```

Comment:
Do you also get the following after the stack dump and the program arguments:

```
1.	While emitting SIL for 'cloneOrFetchDependency' at /tmp/Carthage/Source/CarthageKit/Project.swift:274:10
2.	While emitting reabstraction thunk in SIL function @_TTRXFo_iO11CarthageKit13CarthageError_dT__XFo_oS0__d
```

Makes me think the problem might be due to `CarthageError`, rather than the particulars of this method. :confused: